src/topology/metric_space/basic.lean

```diff
@@ -1254,6 +1254,18 @@ open metric
 class proper_space (α : Type u) [pseudo_metric_space α] : Prop :=
 (compact_ball : ∀x:α, ∀r, is_compact (closed_ball x r))
 
+/-- A proper pseudo metric space is sigma compact, and therefore second countable. -/
+@[priority 100] -- see Note [lower instance priority]
+instance second_countable_of_proper [proper_space α] :
+  second_countable_topology α :=
+begin
+  suffices : sigma_compact_space α, by exactI emetric.second_countable_of_sigma_compact α,
```

Comment:
Why not register the `sigma_compact_space` instance also?

Reply:
We already have https://leanprover-community.github.io/mathlib_docs/topology/subset_properties.html#sigma_compact_space_of_locally_compact_second_countable
I think that adding non-necessary edges to the instance "graph" will increase instance search time (am I wrong?).